dbms/src/Encryption/AESCTRCipherStream.cpp

```diff
@@ -89,13 +98,14 @@ void AESCTRCipherStream::cipher(uint64_t file_offset, char * data, size_t data_s
     ret = EVP_CIPHER_CTX_set_padding(ctx, 0);
     if (ret != 1)
     {
+        FreeCipherContext(ctx);
```

Comment:
Use `SCOPE_EXIT` to free the cipher context?

Reply:
Fixed.